fragments/app-sso-client/accelerator.yaml

```diff
@@ -47,7 +47,7 @@ engine:
               ## Resources needed for AppSSO integration
               
               Please first familiarize yourself with basic concepts around using AppSSO to secure your workload on TAP following 
-              ["Securing your first Workload" tutorial](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-app-sso-app-operators-tutorials-index.html)
+              ["Securing your first Workload" tutorial](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-app-sso-app-operators-tutorials-securing-first-workload.html)
```

Comment:
I wonder if we should link to older TAP docs since we need to provide the TAP version in the link, this would need to be updated for each new TAP release

Reply:
I think all TAP docs links have the version in the them. What is the alternative?